src/Services/WordService.php

```diff
@@ -71,6 +74,9 @@ public static function unsetCurrentWord(): void
         Session::unset('current_word_tip');
```

Comment:
I think Session::unset could receive an array of strings so it could unset things in bulk.